### PR TITLE
mpdecimal, new recipe

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: dev-libs/mpdecimal/mpdecimal-4.0.0.recipe [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+	packageEntries devel \
+		$developDir \
+		$manDir
+}
+
+TEST()
+{
+	export LIBRARY_PATH="$sourceDir/libmpdec${LIBRARY_PATH:+:$LIBRARY_PATH}"
+	export LIBRARY_PATH="$sourceDir/libmpdec++${LIBRARY_PATH:+:$LIBRARY_PATH}"
```
Comment (jmairboeck) [inline dev-libs/mpdecimal/mpdecimal-4.0.0.recipe:78]: maybe put these 2 lines in one? setting the same variable twice looks weird ...

[the remark this answers is not in the document]
Comment (Begasus) [reply dev-libs/mpdecimal/mpdecimal-4.0.0.recipe:78]: Let me do a quick check, thanks!

[the remark this answers is not in the document]
Comment (jmairboeck) [reply dev-libs/mpdecimal/mpdecimal-4.0.0.recipe:78]: Thanks! I would keep the `${...}` expression at the end though.

[the remark this answers is not in the document]
Comment (Begasus) [reply dev-libs/mpdecimal/mpdecimal-4.0.0.recipe]: @jmairboeck like: 
```export LIBRARY_PATH="$sourceDir/libmpdec:$sourceDir/libmpdec++${LIBRARY_PATH:+:$LIBRARY_PATH}"```

[the remark this answers is not in the document]
Comment (jmairboeck) [reply dev-libs/mpdecimal/mpdecimal-4.0.0.recipe]: yes, I think that would be the best. That way it also works if LIBRARY_PATH is empty or not set before.

**Comment by Begasus** [comment]
OK, squashing is giving me a headache, so merging as is with the commits. :)

**Pull request merged**